packages/block-library/src/term-template/use-current-term.js

```diff
@@ -0,0 +1,88 @@
+/**
```

Comment:
This `useCurrentTerm` is very similar to the  `useArchiveLabel`, `useTermDescription`, and `useTermName` hooks, so I think we should consolidate this functionality as much as possible in a follow-up.